node/util_test.go

```diff
@@ -0,0 +1,153 @@
+package node
+
+import (
+	"bytes"
+	"context"
+	"encoding/gob"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/config"
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+// NewNetwork creates a network of fully connected nodes.
+func NewNetwork(conf config.Config, l log.Log, size int) ([]*TestApp, func() error, error) {
```

Comment:
please rename to NewTestNetwork

Reply:
Giving the function `tb testing.TB` as parameter also further clarifies that is only to be used in tests and allows access to `testing.T` in the function 🙂